qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/SnapshotBasedRecoveryIT.java

```diff
@@ -87,6 +123,97 @@ public void testSnapshotBasedRecovery() throws Exception {
         }
     }
 
+    private void updateIndexSettings(String indexName, Settings settings) throws IOException {
```

Comment:
This looks nearly identical to the static `updateIndexSettings` inherited from `ESRestTestCase`, do we need this?